extension/src/validator/authentication-validator.js

```diff
@@ -0,0 +1,44 @@
+const config = require('../config/config')
+
+function isAuthorized(paymentObject, authTokenString) {
+  const ctpProjectKey = paymentObject.custom.fields.commercetoolsProjectKey
+  const isAuthEnabled = _isAuthEnabled(ctpProjectKey)
+
+  console.log(`isAuthEnabled : ${isAuthEnabled}`)
```

Comment:
Remove this log

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/584/commits/b9321da71d385fd3351ad010dcb81ea5a0a0f52c